tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+    [[highlight]]return(new Text(wordPair.asPascalCase));[[/highlight]]
+  }
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Remove the word generation code from MyApp by making the highlighted
+changes below:
+
+<!-- skip -->
+{% prettify dart %}
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    [[strike]]var wordPair = new WordPair.random();[[/strike]]  // Delete this line
+
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+       ),
+        body: new Center(
+          //child: new [[highlight]]Text(wordPair.asPascalCase)[[/highlight]] // Change the highlighted text to...
+          child: new [[highlight]]RandomWords()[[/highlight]] // ... this highlighted text
+        ),
+      ),
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+</ol>
+
+Run the app. It should behave as before, displaying a word
+pairing each time you hot reload or save the app.
+
+### Problems?
+
+If your app isn't running correctly, you can use the code
+below to get back on track.
+
+The **pubspec.yaml** file won't change again.
+The **lib/main.dart** file should now look like this:
+
+{% prettify dart %}
+import 'package:flutter/material.dart';
+import 'package:english_words/english_words.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+        ),
+        body: new Center(
+          child: new RandomWords()
+        ),
+      ),
+    );
+  }
+}
+
+class RandomWords extends StatefulWidget {
+  @override
+  createState() => new RandomWordsState();
+}
+
+class RandomWordsState extends State<RandomWords> {
+  @override
+  Widget build(BuildContext context) {
+    var wordPair = new WordPair.random();
+    return(new Text(wordPair.asPascalCase));
+  }
+}
+{% endprettify %}
+
+---
+
+## Step 4: Create a ListView containing the word pairings
+
+In this step, you'll expand RandomWordsState to generate
+and display a list of word pairings. As the user scrolls, the list
+displayed in a ListView widget, grows infinitely. ListView's
+`builder` factory constructor allows you to build a list view
+lazily, on demand.
+
+<ol markdown="1">
+
+<li markdown="1"> Add a `_suggestions` list to the RandomWordsState
+class for saving suggested word pairings. Note that the variable begins
+with an underscore (`_`).  Prefixing an identifier with an underscore enforces
+privacy in the Dart language.
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  [[highlight]]final _suggestions = <WordPair>[];[[/highlight]]
+  ...
+}
+{% endprettify %}
+</li>
+
+<li markdown="1"> Add a `_buildSuggestions()` function to the RandomWordsState
```

Comment:
Should we use the term "method" when we refer to a function associated with an object?

Reply:
Good point.